README.md

```diff
@@ -1,32 +1,32 @@
 # pelion-edge-provisioner (pep)
```

Comment:
Is this the official name of the tool, signed off by Product? All lower case, with hyphens?